src/app/features/report-expense/report-expense-create-or-edit-dialog/report-expense-create-or-edit-dialog.component.html

```diff
@@ -1,12 +1,57 @@
-<h1 mat-dialog-title>Hi {{data.name}}</h1>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+<h1 mat-dialog-title>Add Account {{data.name}}</h1>
 <div mat-dialog-content>
-  <p>What's your favorite animal?</p>
-  <mat-form-field appearance="fill">
-    <mat-label>Favorite Animal</mat-label>
-    <input matInput [(ngModel)]="data.animal">
+  <mat-form-field class="date-picker" appearance="outline">
```

Comment:
@yosefmare 
As you can see in the attached photo the `Date` field is different in the size from the other fields, can you please resize it to fit to the others?
<img width="653" alt="image" src="https://user-images.githubusercontent.com/12885037/193424783-0c91d517-e647-44d0-adb2-a6f56c4ab25c.png">

Reply:
[
![image](https://user-images.githubusercontent.com/106099154/193454463-bdb97dcf-68dd-4a08-8623-7cbe972a8bf3.png)
](url)

do you think the buttons looks good like that?

Reply:
[
![image](https://user-images.githubusercontent.com/106099154/193454593-624201b8-ec6e-4ea1-9897-bea74570542f.png)
](url)
do you think the b buttons looks good like that?